src/plugins/_index.scss

```diff
@@ -0,0 +1,6 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
```

Comment:
Shouldn't this be `src/plugins/eui_utils/public/sass/_styling_constants.scss`?

Reply:
yes

Reply:
updated